src/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import Home from "./pages/Home";
+import Video from "./pages/Video";
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
```

Comment:
Move library imports above relative imports

Reply:
Updated to be separated by newlines.